### PR TITLE
Adding Whats new for iOS

### PR DESCRIPTION
**Pull request by sunasing**

### CODE REVIEW

**Comment by sunasing** [comment]
#sign-off

**Comment by sunasing** [comment]
Can we include this in the TOC: 

> Microsoft Defender for Endpoint for iOS

> > Overview
> > **What's new**

**Comment by sunasing** [comment]
#sign-off

File: windows/security/threat-protection/microsoft-defender-atp/ios-whatsnew.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# What's new in Microsoft Defender for Endpoint for iOS
+
+[!INCLUDE [Microsoft 365 Defender rebranding](../../includes/microsoft-defender.md)]
+
+**Applies to:**
+- [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/p/?linkid=2146631)
+- [Microsoft 365 Defender](https://go.microsoft.com/fwlink/?linkid=2118804)
+
+> Want to experience Microsoft Defender for Endpoint? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-exposedapis-abovefoldlink)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-whatsnew.md:31]:
```suggestion
Want to experience Microsoft Defender for Endpoint? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-exposedapis-abovefoldlink)
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-whatsnew.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## 1.1.15010101
+
+- With this version, we are announcing support for iPadOS / iPad devices.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-whatsnew.md:35]:
```suggestion
- With this version, we are announcing support for iPadOS/iPad devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by illfated — APPROVED** [review]
Looks good to me, with JohanFreelancer9's suggestions added.

**Pull request merged**